bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java

```diff
@@ -34,7 +34,7 @@
  * version, is bound to this proxy component resource type.
  * </p>
  */
-@Version("5.2.0")
```

Comment:
@rismehta / @ujjwal5 Is this some typo, or it is intentional to keep package info version higher than master?